catroid/src/main/java/org/catrobat/catroid/content/bricks/SetBackgroundAndWaitBrick.java

```diff
@@ -36,17 +37,18 @@
 
 public class SetBackgroundAndWaitBrick extends SetBackgroundBrick {
 
-	public SetBackgroundAndWaitBrick() {
-	}
+	public SetBackgroundAndWaitBrick() { }
```

Comment:
this is not correctly formatted. I wonder what's going on with our static checks here, how did they miss it?

Reply:
Looks like a bug to me → reported